site2/website-next/src/pages/index.js

```diff
@@ -177,14 +177,10 @@ export default function Home() {
         </div>
         <div className="home-ctas relative z-5">
           <div className="inner">
-            <PillButton
-              variant=""
-              target=""
-              href={docUrl("next/concepts-overview")}
-            >
+            <PillButton variant="" target="" href={docUrl("concepts-overview")}>
               Learn More
             </PillButton>
-            <PillButton variant="grey" href={docUrl("next/")}>
+            <PillButton variant="grey" href={docUrl("getting-started-home")}>
```

Comment:
```suggestion
            <PillButton variant="grey" href={docUrl("getting-started-standalone")}>
```

After the latest stable version becomes 2.11.x, we can revert this change.